### PR TITLE
Sam/aq client

### PR DESCRIPTION
**Pull request by samholmes**
#### PR Dependencies

 none

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202824923939286

### CODE REVIEW

File: src/controllers/action-queue/push.js [hunk 1]
```diff
@@ -88,11 +86,12 @@ export async function prepareNewPushEvents(
...
 }
 
-export async function checkPushEvents(account: EdgeAccount, eventIds: string[]): Promise<boolean> {
+export async function checkPushEvents(context: ExecutionContext, eventIds: string[]): Promise<boolean> {
+  const { account, clientId } = context
   const { rootLoginId: loginId } = account
   const requestBody: PushRequestBody = {
     apiKey: AIRBITZ_API_KEY,
-    deviceId,
+    deviceId: clientId,
```
Comment (Jon-edge) [inline src/controllers/action-queue/push.js:94]: Why was PushRequestBody not updated to clientId?

**Review by Jon-edge — APPROVED** [review]
Approved after the one comment is justified or changed

File: src/controllers/action-queue/push.js (hunk 1 above)
Comment (samholmes) [reply src/controllers/action-queue/push.js:94]: `deviceId` is apart of the push-server API spec. Changing this requires coordination with the push-server implementation. Now may be the time to push for such a change, but we can also be lazy here haha

**Review by swansontec — APPROVED** [review]

**Comment by samholmes** [comment]
Redo PR: https://github.com/EdgeApp/edge-react-gui/pull/3614

**Pull request merged**